docs/reference/persistent-queues.md

```diff
@@ -81,6 +81,11 @@ If you want to define values for a specific pipeline, use [`pipelines.yml`](/ref
 
     To avoid losing data in the persistent queue, you can set `queue.checkpoint.writes: 1` to force a checkpoint after each event is written. Keep in mind that disk writes have a resource cost. Setting this value to `1` ensures maximum durability, but can severely impact performance. See [Controlling durability](#durability-persistent-queues) to better understand the trade-offs.
 
+`queue.compression`
+:   Sets the event compression goal for use with the persisted queue. Default is `none`. Acceptable values include:
+    * `speed`: optimize for fastest compression operation
+    * `size`: optimize for smallest size on disk, spending more CPU
+    * `balanced`: a balance between the `speed` and `size` settings
```

Comment:
Can we provide some guidance here on what the users should be able to see in their metrics to decide when to turn this on?

For example, we provide an example of observing mmap operations on multiple consecutive api calls to hot_threads. or/and a combination of low worker utilization and high queue backpressure.

Reply:
I'll be addressing this with #18107